core/src/main/java/org/arquillian/smart/testing/SystemPropertyResolver.java

```diff
@@ -0,0 +1,33 @@
+package org.arquillian.smart.testing;
+
+public class SystemPropertyResolver {
```

Comment:
yeah currently it's resolving only git properties, but in future it might be possible we can add more system properties which is not for git. So we can easily add here. instead of creating different class for System property using category it's good to have everything at one place.